regression/scripts/airfoil_analysis/airfoil_panel_method_test.py

```diff
@@ -0,0 +1,209 @@
+# airfoil_panel_method_test.py
+# 
+# Created:  
+# Modified: Mar 2021, M. Clarke 
+
+#----------------------------------------------------------------------
+#   Imports
+# ---------------------------------------------------------------------
+
+import SUAVE 
+import os
+from SUAVE.Core import Units, Data 
+from SUAVE.Methods.Aerodynamics.Airfoil_Panel_Method.airfoil_analysis import airfoil_analysis
+import matplotlib.pyplot as plt   
+from SUAVE.Methods.Geometry.Two_Dimensional.Cross_Section.Airfoil.compute_naca_4series \
+     import  compute_naca_4series
+from SUAVE.Plots.Airfoil_Plots  import plot_airfoil_properties
+
+import numpy as np
+
+# ----------------------------------------------------------------------
+#   Main
+# ----------------------------------------------------------------------
+
+def main():    
+    # Define Panelization 
+    npanel = 160
+    
+    # Define Reynolds Number
+    Re     = 5E6 
+    
+    #Define Angle of Attack
+    AoA1   = 3*Units.degrees 
```

Comment:
It would be very useful if this could be a vectorized input

Reply:
*partially* vectoried